main/musl-mallocng/patches/getdents.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/dirent/posix_getdents.c b/src/dirent/posix_getdents.c
```

Comment:
any reason mallocng has a separate patch folder?

Reply:
because the main patches won't apply unless compiler-rt sources are present

Reply:
(the alternative is to unify sources for mallocng and always apply scudo into the codebase, which is actually maybe better? idk, what do you think?)

Reply:
hm right

i think that is better since then we effectively have identical templates that differ only in a configure arg. or at least it seems better to me since there's less diff

Reply:
pushed what i think is correct, should be the same now. i didn't test it at runtime though..

Reply:
ok, tested. stuff uhh still works, and based on build log it's still mallocng and not scudo, looks fine

Reply:
looks good :)

Reply:
thanks for looking!